networks/osmosis-1/upgrades/v18/v18_binaries.json

```diff
@@ -1,6 +1,8 @@
 {
     "binaries": {
-        "linux/arm64": "https://github.com/osmosis-labs/osmosis/releases/download/v18.0.0/osmosisd-18.0.0-linux-arm64?checksum=sha256:4331f9a318f6dd2f012c36f6ef19af8378fd1e9bc85c751e3f56f7018176ed58",
-        "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v18.0.0/osmosisd-18.0.0-linux-amd64?checksum=sha256:9a98a57946e936e7380ae897a205b4e18a188332e91ca84a1f62c21cbb437845"
+        "darwin/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v18.0.0/osmosisd-18.0.0-darwin-amd64?checksum=sha256:218f230a2418faab7b980a312a59e7b38a28eab8e3a857c46d78ab7ce10a7aaa",
+        "darwin/arm64": "https://github.com/osmosis-labs/osmosis/releases/download/v18.0.0/osmosisd-18.0.0-darwin-arm64?checksum=sha256:31280f32538e4e353d1a68adff6219c197ab63166faaca6eced9114c90a5a5f9",
```

Comment:
Can we actually remove darwin from this release, since it won't be state machine compatible?

We can test darwin in a patch release (happy to just make a patch release that does nothing aside from adding darwin here)